ui/src/app/components/apps-panel/utils.tsx

```diff
@@ -168,3 +174,35 @@ export const toUserEnvironmentStatusByAppType = (
 export const showAppsPanel = (config: ConfigResponse) => {
   return config.enableCromwellGKEApp || config.enableRStudioGKEApp;
 };
+
+export interface AppState {
```

Comment:
How about App**Display**State or similar?  I expect this will get confused with app **statuses** like Running, Error, etc.

Reply:
Good call! Thanks for the suggestion.